src/main/kotlin/no/nav/familie/ks/sak/kjerne/beregning/TilkjentYtelseValidator.kt

```diff
@@ -34,7 +34,10 @@ object TilkjentYtelseValidator {
         val søker = personopplysningGrunnlag.søker
         val barna = personopplysningGrunnlag.barna
 
-        val diff = Period.between(tilkjentYtelse.stønadFom?.toLocalDate(), tilkjentYtelse.stønadTom?.toLocalDate())
+        val stønadFom = tilkjentYtelse.andelerTilkjentYtelse.minOf { it.stønadFom }
```

Comment:
Hva skjer dersom andelerTilkjentYtelse er en tom liste? Det er kanskje ikke mulig å komme hit dersom det ikke finnes noen andelerTilkjentYtelse?

Reply:
@bragejahren god poeng. Jeg tror det kan faktisk hende for Fortsatt Innvilget tilfeller. Derfor lagde vi en if sjekk på andelTilkjentYtelse før vi validerer dette